src/analyze/FindFiles.re

```diff
@@ -146,8 +146,16 @@ let filterDuplicates = cmts => {
 let nameSpaceToName = n => n |> Str.split(Str.regexp_string("-")) |> List.map(String.capitalize_ascii) |> String.concat("");
 
 let getNamespace = config => {
-  let isNamespaced = Json.get("namespace", config) |?> Json.bool |? false;
-  isNamespaced ? (config |> Json.get("name") |?> Json.string |! "name is required if namespace is true" |> nameSpaceToName |> s => Some(s)) : None;
+  let ns = Json.get("namespace", config);
```

Comment:
Could you make this backwards-compatible? So if the bs-platform version is pre-5.0.0, it requires it to be a boolean, otherwise it can be a string? (that would involve `getNamespace` taking a `~supportsNamespaceRename` parameter or something)

Reply:
would reading the bs version from `package.json` be acceptable?